server/backbone/api/parent.py

```diff
@@ -542,3 +558,70 @@ def get_child_quests(cid, start, lookahead):
     if not child_is_my_child(child):
         raise BackboneException(404, "Child not found")
     return json_return(get_childs_quest_with_window(start, lookahead))
+
+
+@api_bp.route('/quest/<int:qid>/verify', methods=['POST'], defaults={'ts': None})
+@api_bp.route('/quest/<int:qid>/<float:ts>/verify', methods=['POST'])
+@parent_login_required
+@backbone_error_handle
+def verify_quest_completion(qid, ts):
```

Comment:
we also need `reject_quest_completion` or something so the parent can reject the proof. Then the chore should be updated with `completed_now: false` so it becomes an upcoming chore again instead of pending

Reply:
`completed_now` isn't used or saved anywhere, it just tells the app that the call that it just made was the one that completed the quest, so if you try to complete, or verify the quest more than once, the app can tell that the last call didn't do anything.

Reply:
sorry I meant `completed_on` then

Reply:
Can we decide on the reject next week? I'll do it then, but if you want this before then we can merge this separately.

Reply:
ya can do it later